features/partners.feature

```diff
@@ -274,7 +276,7 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
       | met onbekende einddatum | 00000000         | 555550003           | Osama     |
 
   @proxy
-  Rule: Wanneer geen van de gegevens van de partner een waarde heeft, wordt de partner geleverd met type "PartnerOnbekend" en indicatieOnbekend met waarde true
+  Rule: Wanneer geen van de gegevens van de partner een waarde heeft, wordt er een 'partners' zonder gegevens geleverd.
     - Dit is het geval wanneer elk gegeven van de partner leeg is of een standaardwaarde heeft
```

Comment:
"Dit is het geval wanneer elk **gevraagd** gegeven ..."

Reply:
aangepast